solr/core/src/java/org/apache/solr/handler/SolrConfigHandler.java

```diff
@@ -255,11 +256,17 @@ private void handleGET() {
                   for (PackageListeners.Listener listener :
                       listeners) {
                     PluginInfo info = listener.pluginInfo();
-                    if(info == null) continue;
+                    if(info == null) {
+                      log.info("listener.pluginInfo was null for {}; continuing", componentName);
+                      continue;
+                    }
                     if (info.type.equals(parts.get(1)) && info.name.equals(componentName)) {
-                      if (o instanceof Map) {
```

Comment:
o was not an instanceof Map, and I'm not sure why - PluginInfo implements MapSerializable which has a toMap function. This needs fixing, but for immediate debugging of underlying plugin issues, just logging the version lets me move ahead while this is still to-be-fixed.